depends/packages/gmp.mk

```diff
@@ -6,6 +6,8 @@ $(package)_sha256_hash=5275bb04f4863a13516b2f39392ac5e272f5e1bb8057b18aec1c9b79d
 
 define $(package)_set_vars
 $(package)_config_opts+=--enable-cxx --enable-fat --with-pic --disable-shared
+$(package)_cflags_armv7l_linux+=-march=armv7-a
+$(package)_cppflags_armv7l_linux+=-march=armv7-a
```

Comment:
are you sure that you meant `cppflags` and not `cxxflags`?

Reply:
Good point, I think you are right. I tried again with `cxxflags` instead of `cppflags`, and one more time deleting the entire line and only specifying `cflags`. In both cases it builds without errors. gmp sets the following build environment for itself when running `./configure` separately (outside the depends system):
```
using ABI="32"
      CC="gcc"
      CFLAGS="-O2 -pedantic -fomit-frame-pointer -march=armv8-a -mfloat-abi=hard -mfpu=neon -mtune=cortex-a53"
      CPPFLAGS=""
      MPN_PATH=" arm/neon arm/v7a/cora9 arm/v6t2 arm/v6 arm/v5 arm generic"
```

Raspberry Pi 3 B is technically an ARMv8 processor, but I stepped the requirement down to ARMv7 to continue supporting Raspberry Pi 2. I omitted the other flags since they were not required to build without errors.